### DISCUSSION TITLE
Manual/class-AudioClip

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#155] [Notes]
# Manual/class-AudioClip

https://docs.unity3d.com/Manual/class-AudioClip.html

**Comment by maizedw** [comment]
For a much more detailed explanation of different import settings for audio and their effects, see [Unity Audio Import Optimisation - getting more BAM for your RAM](https://www.gamedeveloper.com/audio/unity-audio-import-optimisation---getting-more-bam-for-your-ram).
